acls_test.go

```diff
@@ -52,6 +59,244 @@ func (s *Suite) TestBasicRule(c *check.C) {
 	c.Assert(rules, check.NotNil)
 }
 
```

Comment:
```suggestion

# TODO(kradalby): Make tests values safe, independent and descriptive.
```

Lets add this so I remember, I think its easier for me to show, rather than trying to explain, because I am not that good at formulating these arguments.

Reply:
oh wops